bot.user.js

```diff
@@ -1076,6 +1076,23 @@ var userInterface = (function() {
                     'green;\">enabled' : 'red;\">disabled') +
                 '</span>';
         }
+        
+        // Set high score
+    setHighScore: function(){
```

Comment:
This error comes from a missing comma on line 1078.

This also threw off your IDE's indentation... line 1080 is correct but 1081 onward needs an extra four spaces.